software/base/src/main/java/brooklyn/entity/software/StaticSensor.java

```diff
@@ -36,9 +43,17 @@ public StaticSensor(ConfigBag params) {
         value = params.get(STATIC_VALUE);
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     public void apply(EntityLocal entity) {
         super.apply(entity);
-        entity.setAttribute(sensor, (T) TypeCoercions.coerce(value, sensor.getType()));
+        
+        Maybe<T> v = Tasks.resolving(value).as((Class<T>)sensor.getType()).timeout(Duration.millis(200)).getMaybe();
```

Comment:
Why not set the raw value and resolve & coerce at read time?

The resolve timeouts seem to be arbitrarily chosen - I've seen 50ms at other places. Better use a global constant instead?